main.js

```diff
@@ -0,0 +1,115 @@
+// book constructor
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+// storage
+function getBook() {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
+  } else {
+    books = JSON.parse(localStorage.getItem('books'));
+  }
+
+  return books;
+}
+
+function addBook(book) {
+  const books = getBook();
+  books.push(book);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function removeBook(title) {
+  const books = getBook();
+  books.forEach((book, index) => {
+    if (book.title === title) {
+      books.splice(index, 1);
+    }
+  });
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function addBookToList(book) {
+  const list = document.querySelector('#book-list');
+
+  const row = document.createElement('tr');
+  row.innerHTML = `
+    <td>${book.title}</td>
+    <td>${book.author}</td>
+    <td>${book.pages}</td>
+    <td>${book.read}</td>
+    <td><a href="#" class="delete">X</a></td>
+    `;
+
+  list.appendChild(row);
```

Comment:
- Keep application logic separate from DOM manipulation logic. Move this code to another file.